src/test/java/api/ItemsByInstanceResourceTest.java

```diff
@@ -77,6 +78,38 @@ void canGetInstanceById() {
       hasJsonPath("tenantId", is(TENANT_ID_UNIVERSITY)))));
   }
 
+  @Test
+  void canGetEmptyResult() {
+    IndividualResource instance = instancesFixture.basedUponDunkirk();
+    UUID instanceId = instance.getId();
+
+    // create item in tenant "college"
+    setTempTenantId(TENANT_ID_COLLEGE);
+    IndividualResource collegeLocation = locationsFixture.mainFloor();
+    IndividualResource collegeHoldings = holdingsFixture.defaultWithHoldings(instanceId);
+    IndividualResource collegeItem = itemsFixture.createItemWithHoldingsAndLocation(
+      collegeHoldings.getId(), collegeLocation.getId());
+    clearTempTenantId();
+
+    // create item in tenant "university"
+    setTempTenantId(TENANT_ID_UNIVERSITY);
+    IndividualResource universityLocation = locationsFixture.thirdFloor();
+    IndividualResource universityHoldings = holdingsFixture.defaultWithHoldings(instanceId);
+    IndividualResource universityItem = itemsFixture.createItemWithHoldingsAndLocation(
+      universityHoldings.getId(), universityLocation.getId());
+    clearTempTenantId();
+
+    // make sure neither item exists in current tenant
+    assertThat(itemsFixture.getById(collegeItem.getId()).getResponse().getStatusCode(), is(404));
+    assertThat(itemsFixture.getById(universityItem.getId()).getResponse().getStatusCode(), is(404));
+
+    ResourceClient.forSearchClient().replace(instanceId, new JsonObject());
+    Response response = get(String.format("query=(id==%s)", instanceId), 200);
+    JsonObject responseJson = response.getJson();
+
+    Assertions.assertTrue(responseJson.isEmpty());
```

Comment:
I would suggest using the same import for assertions as seen throughout this test

Reply:
Thanks for noticing it, fix it